x-pack/plugins/endpoint/common/types.ts

```diff
@@ -44,3 +62,37 @@ export interface EndpointMetadata {
     };
   };
 }
+
+export type AlertData = Immutable<{
```

Comment:
Why should this be `Immutable`?

Reply:
There should be one section of the alert data that is mutable... otherwise, it should all be immutable.

Reply:
done